clio-client/src/main/scala/org/broadinstitute/clio/client/ClioClient.scala

```diff
@@ -64,26 +65,39 @@ object ClioClient extends LazyLogging {
         sys.exit(1)
       }
 
+    def early(source: ClioClient.EarlyReturn): Unit = {
+      source match {
+        case UsageOrHelpAsked(message) => {
+          println(message)
+          sys.exit(0)
+        }
+        case ParsingError(error) => {
+          System.err.println(error.message)
+          sys.exit(1)
+        }
+      }
+    }
+
+    def complete(done: Try[Done]): Unit = {
+      val status = done match {
+        case Success(_) => 0
+        case Failure(ex) =>
+          logger.error("Failed to execute command", ex)
+          1
+      }
+      Await.result(system.terminate(), 23.seconds)
```

Comment:
It looks like this is the only change here, is that correct?

Maybe a comment would be helpful, at least for the magic number.

Reply:
I thought the magic number was magic enough for its value to be obviously arbitrary, but OK.